testing/features/support/components/search/standard.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Search
+  class Standard < ::Base
+    set_url "/iframe.html?id=components-search--search&viewMode=story"
+
+    section :cads_search, ".cads-search" do
+      element :search_field, ".cads-search__input"
+      element :search_button, ".cads-search__button"
+    end
+
+    def validate_initial_state!
+      has_cads_search?(wait: 5)
+    end
+
+    def clear_field
+      cads_search.search_field.value.length.times do
+        cads_search.search_field.send_keys [:backspace]
+      end
+    end
```

Comment:
I tried:
```
@component.cads_search.search_field.send_keys([:command, 'a'], :backspace)
```
But this way was not going to work for windows since it is `[:control, 'a']`

Reply:
I've no issue with this, but it's a bit clunky / heavy. A quicker one would be a native double click. This highlights all.

So something like
```
search_field.double_click
search_field.send_keys(:delete)
```

Reply:
The problem with double_click is that the pre-defined value has more than one word, so the double_click just selects the last one.
<img width="712" alt="Screenshot 2020-11-27 at 18 12 05" src="https://user-images.githubusercontent.com/11262774/100476072-91ae7500-30dc-11eb-9887-ab47e80e0cb0.png">
<img width="708" alt="Screenshot 2020-11-27 at 18 12 17" src="https://user-images.githubusercontent.com/11262774/100476085-97a45600-30dc-11eb-8245-508c8167c6f7.png">